<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_exportFbxTools</class>
 <widget class="QWidget" name="form_exportFbxTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>166</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Export FBX Tools v1.0</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="2" column="0" colspan="7">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="7">
       <widget class="QPushButton" name="pushButton_export">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #707070; /* 默认背景颜色 */
	border-width: 0px;
	border-radius: 15px;
}

QPushButton:hover {
    background-color: rgba(0,0,200,60); /* 鼠标悬停颜色 */
}

QPushButton:pressed {
    background-color: rgba(0,0,0,180); /* 鼠标点击颜色 */
}
</string>
        </property>
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QRadioButton" name="radioButton_otherPart">
        <property name="text">
         <string>Other</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="7">
       <widget class="QPushButton" name="pushButton_exit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #707070; /* 默认背景颜色 */
	border-width: 0px;
	border-radius: 15px;
}

QPushButton:hover {
    background-color: rgba(0,0,200,60); /* 鼠标悬停颜色 */
}

QPushButton:pressed {
    background-color: rgba(0,0,0,180); /* 鼠标点击颜色 */
}
</string>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QRadioButton" name="radioButton_headPart">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Head</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Export Part : </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="5">
       <widget class="QLineEdit" name="lineEdit_fbxPath">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="5">
       <widget class="QLineEdit" name="lineEdit_fbxName">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QRadioButton" name="radioButton_bodyPart">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Body</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="pushButton_loadfbxPath">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #707070; /* 默认背景颜色 */
	border-width: 0px;
	border-radius: 15px;
}

QPushButton:hover {
    background-color: rgba(0,0,200,60); /* 鼠标悬停颜色 */
}

QPushButton:pressed {
    background-color: rgba(0,0,0,180); /* 鼠标点击颜色 */
}
</string>
        </property>
        <property name="text">
         <string>&lt;&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Export Name : </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Export FBX Path : </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="7">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_exit</sender>
   <signal>clicked()</signal>
   <receiver>form_exportFbxTools</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
